10.9-libcxx/stable/main/finkinfo/libs/pythonmods/sphinxcontrib-websupport-py.info

```diff
@@ -0,0 +1,43 @@
+Info2: <<
+
+Package: sphinxcontrib-websupport-py%type_pkg[python]
+Version: 1.1.0
+Revision: 1
+License: OSI-Approved
+Type: python (2.7 3.4 3.5 3.6 3.7)
+
+Depends: python%type_pkg[python]
+BuildDepends: setuptools-tng-py%type_pkg[python]
+
+Source: https://files.pythonhosted.org/packages/source/s/sphinxcontrib-websupport/sphinxcontrib-websupport-%v.tar.gz
+Source-MD5: ca6435e7b4eb9408df4f54972361e9d3
+
+CompileScript: %p/bin/python%type_raw[python] setup.py build
+
+InstallScript: %p/bin/python%type_raw[python] setup.py install --root=%d
+
+DocFiles: CHANGES LICENSE PKG-INFO README.rst
+
+InfoTest: <<
+  TestDepends: <<
+    mock-py%type_pkg[python],
+    pytest-py%type_pkg[python],
+    sqlalchemy-py%type_pkg[python],
+    tox-py%type_pkg[python],
+    whoosh-py%type_pkg[python],
+    (%type_pkg[python]  = 27) xapian22-py%type_pkg[python],
+    (%type_pkg[python] >= 34) xapian30-py%type_pkg[python]
+  <<
+  TestScript: %p/bin/python%type_raw[python] setup.py test || exit 2
+  # The test above does nothing, as does nosetests.  The following trials
+  # also fail and i have no clue how to fix it.
+  # - %p/bin/python%type_raw[python] -m pytest
+  # - tox
```

Comment:
Have you tried `%p/bin/python%type_raw[python] -m pytest tests/ || exit 2` for running the tests?